lib/bech32/src/Codec/Binary/Bech32/Internal.hs

```diff
@@ -14,18 +14,19 @@
 --
 -- From an original implementation by Marko Bencun:
 --
--- [sipa/bech32](https://github.com/sipa/bech32/tree/bdc264f84014c234e908d72026b7b780122be11f/ref/haskell)
-
+-- [sipa/bech32](https://git.io/fj8FV)
```

Comment:
These shortened URLs are an official part of GitHub (and are easier to paste into source code).